code/__DEFINES/construction.dm

```diff
@@ -108,6 +108,8 @@
 #define RCD_MACHINE 8
 #define RCD_COMPUTER 16
 
-#define RCD_UPGRADE_FRAMES	1
-#define RCD_UPGRADE_SIMPLE_CIRCUITS	2
-#define RCD_UPGRADE_SILO_LINK	4
+#define RCD_UPGRADE_FRAMES	(1<<0)
+#define RCD_UPGRADE_SIMPLE_CIRCUITS	(1<<1)
+#define RCD_UPGRADE_SILO_LINK	(1<<2)
+
+#define RPD_UPGRADE_UNWRENCH (1<<0)
```

Comment:
Why is this using the same bitshift as `RCD_UPGRADE_FRAMES`? Shouldn't it be 1<<3?

Reply:
Uhm... Because you cannot apply an RCD upgrade onto an RPD or vice-versa? This also allows for more RPD upgrades in the future if that ever happens without limiting RCD upgrades.

Reply:
Oh. Misread.